Sources/Features/Metrics/Model/Payload.swift

```diff
@@ -0,0 +1,33 @@
+import Foundation
+
+struct EventsMetadata: Encodable {
+
+    var droppedEvents = 0
```

Comment:
Should it be mutable? If so, it's fine. Otherwise, please use the `let` keyword. It's minor but it using immutable data types creates fewer bugs, so I'd prefer immutable properties in the first place and make it mutable if and only if it needs to be.

Reply:
This one does need to be mutable. In the future droppedEvents will increment on retry